src/rdposix.h

```diff
@@ -30,7 +29,8 @@
 /**
  * POSIX system support
  */
-#pragma once
+#ifndef RDPOSIX_H
```

Comment:
I got confused too I though you wanted #define <FILENAME> with s/./_/g and tr [a-z] [A-Z]
but now I'm getting that you want to keep the _ prefix I originally put _and_ add a _ for rd modules iow rd\[^.]*\)\.h -> _RD_\1_H (modulo uppercase) 
correct ?